src/goto-instrument/replace_function_bodies.h

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************\
+
+Module: Replace bodies of goto functions
+
+Author: DiffBlue Limited.
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_INSTRUMENT_REPLACE_FUNCTION_BODIES_H
+#define CPROVER_GOTO_INSTRUMENT_REPLACE_FUNCTION_BODIES_H
+
+#include <regex>
+#include <vector>
+#include <iostream>
+
+#include <goto-programs/goto_functions.h>
+#include <util/std_code.h>
+#include <util/std_types.h>
+#include <util/message.h>
+
+/// Replace function bodies with some default behavior
+/// Can generate the following types of function bodies:
+///
+/// assert-false: { __CPROVER_assert(false); }
+///
+/// assume-false: { __CPROVER_assume(false); }
+///
+/// assert-false-assume-false: {
+///   __CPROVER_assert(false);
+///   __CPROVER_assume(false); }
+///
+/// havoc[,params:<regex>][,globals:<regex>]:
```

Comment:
`<regex>` needs some kind of escaping to make Doxygen happy.

Reply:
I think I'll just remove the angle brackets rather than escape, for sake of legibility in code